bin/liquidator/src/rebalance.rs

```diff
@@ -565,4 +638,42 @@ impl Rebalancer {
 
         Ok(())
     }
+
+    fn is_lst(&self, mint: Pubkey) -> bool {
+        self.lst_mints.contains(&mint)
+    }
+
+    pub async fn init(&mut self, live_rpc_client: &RpcClient) {
+        if let Err(e) = self.load_lst_if_needed(live_rpc_client).await {
+            warn!("Could not load list of sanctum supported mint: {}", e);
+        }
+    }
+
+    async fn load_lst_if_needed(&mut self, live_rpc_client: &RpcClient) -> anyhow::Result<()> {
+        if !self.lst_mints.is_empty() {
+            return Ok(());
+        }
+
+        let address = Pubkey::from_str("EhWxBHdmQ3yDmPzhJbKtGMM9oaZD42emt71kSieghy5")?;
+
+        let lookup_table_data = live_rpc_client.get_account(&address).await?;
+        let lookup_table = AddressLookupTable::deserialize(&lookup_table_data.data())?;
+        let accounts: Vec<Account> = live_rpc_client
+            .get_multiple_accounts(&lookup_table.addresses)
```

Comment:
does this emit a single GetMultipleAccounts call? those usually have an upper limit of usually 100 accounts to be fetched at once. Since ALTs can have 255 accounts, this needs to call a helper that splits it up into multiple rpc calls (maybe this function already does it, not sure)

Reply:
It does only one call, I'll try to find the helper if it already exists, or add one :+1: